src/main/java/com/conveyal/gtfs/error/NewGTFSErrorType.java

```diff
@@ -43,6 +43,7 @@
     NUMBER_TOO_SMALL(Priority.MEDIUM, "Number was below the allowed range."),
     OVERLAPPING_TRIP(Priority.MEDIUM, "Blocks?"),
     REFERENTIAL_INTEGRITY(Priority.HIGH, "This line references an ID that does not exist in the target table."),
+    MULTI_REFERENTIAL_INTEGRITY(Priority.HIGH, "This line references an ID that does not exist in a foreign table."),
```

Comment:
Did you mean "multiple foreign tables"?

Reply:
I'm going to rename this as `MULTI_REFERENTIAL_INTEGRITY` is a bit misleading, but does have an important difference to `REFERENTIAL_INTEGRITY`. What I'm trying to get across is that the ID does not reference an ID in one of multiple tables. E.g. `Stop_time.stop_Id` can be either a `Stop.stop_id` OR `Location.location_id`.

Reply:
Updated to `MISSING_FOREIGN_TABLE_REFERENCE`